### PR TITLE
updates to styles, custom.js, and voice errors

### PR DESCRIPTION
**Pull request by dtolb**
Last few stylist and prose fix for LA

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-la-updates-2019-07-31.s3-website-us-east-1.amazonaws.com/

**Review by jchavez443 — APPROVED** [review]

**Review by jmulford-bw — APPROVED** [review]

File: book.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "plugins": [
-        "theme-bandwidth@1.9.11",
+        "theme-bandwidth@1.9.13",
```
Comment (jmulford-bandwidth) [inline book.json]: Should it be .12? Based on https://github.com/Bandwidth/docs-theme/pull/58/files

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: book.json (hunk 1 above)
Comment (dtolb) [reply book.json]: had a 🐛 it got bumped to `.13` :(

Comment (jmulford-bandwidth) [reply book.json]: cool

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**